jscomp/runtime/js.ml

```diff
@@ -41,6 +64,8 @@ type +'a nullable = 'a Js_null_undefined.t
 
 type +'a null_undefined = 'a nullable
 
+external unsafe_downgrade : 'a t -> 'a = "#unsafe_downgrade"
```

Comment:
continuing conversation from https://github.com/melange-re/melange/pull/798#discussion_r1366950903 because I deleted the `.mli` file.

I think we should expose this in a private module and explain why. I'll add a commit and we can see if you agree.

Reply:
What do you think about ba8352954 ?